test/objectFifo-stateful-transform/bd_chain_on_memtile/chain_with_split_and_join.mlir

```diff
@@ -0,0 +1,186 @@
+//===- bd_chain_on_memtile/ichain_with_split_and_join.mlir ----------------*- MLIR -*-===//
```

Comment:
For maintainability and readability, I think it would be good to also add two other tests for each pattern separately, one for distribute and one for join.